### ISSUE TITLE
Gradio 5.0.1 - Gradio.MultimodalTextbox with "image" doesn't work as expected for webp

### ISSUE THREAD

**Issue by skye0402** [gradio-app/gradio#9686]
### Describe the bug

I'm currently upgrading from latest Gradio 4.44 to 5.0.1. The working existing code doesn't work with the way the MultimodalTextbox (MMTB) worked before.

I've set:

```python
msg_box = gr.MultimodalTextbox(
	interactive=True, 
	file_types=["images"],
	file_count="multiple",
	scale=9,
	elem_id="msg_box",
	max_lines=5,
	placeholder="Enter text or images and text and press ENTER",
	container=False,
	autofocus=True
)
```

Which works in 4.44. Now in 5.0.1 if I drag & drop a `webp` file into the MMTB, it doesn't work. The code throws an error:

> gradio.exceptions.Error: "Invalid file type: image/webp. Please upload a file that is one of these formats: ['image']"

The source of the problem seems to be the Python function used to "guess" the mime type. So one could say "not a gradio problem" - but since it worked before it was obviously more tolerant. Besides, `webp `is a very common image format standard.

If found this [Python Issue](https://github.com/google-gemini/generative-ai-python/issues/425) and I upgraded to Python 3.12.7 which is the latest but still not working. I don't want to go Python 3.13 as many AI frameworks are not supporting it yet.

Can you make the MMTB work again please?

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Just using the example with the parameters "image". Drop a webp file into it and press enter.

```python
import gradio as gr
import time

# Chatbot demo with multimodal input (text, markdown, LaTeX, code blocks, image, audio, & video). Plus shows support for streaming text.


def print_like_dislike(x: gr.LikeData):
    print(x.index, x.value, x.liked)


def add_message(history, message):
    for x in message["files"]:
        history.append({"role": "user", "content": {"path": x}})
    if message["text"] is not None:
        history.append({"role": "user", "content": message["text"]})
    return history, gr.MultimodalTextbox(value=None, interactive=False)


def bot(history: list):
    response = "**That's cool!**"
    history.append({"role": "assistant", "content": ""})
    for character in response:
        history[-1]["content"] += character
        time.sleep(0.05)
        yield history


with gr.Blocks() as demo:
    chatbot = gr.Chatbot(elem_id="chatbot", bubble_full_width=False, type="messages")

    chat_input = gr.MultimodalTextbox(
        interactive=True,
        file_count="multiple",
        file_types=["image"],
        placeholder="Enter message or upload file...",
        show_label=False,
    )

    chat_msg = chat_input.submit(
        add_message, [chatbot, chat_input], [chatbot, chat_input]
    )
    bot_msg = chat_msg.then(bot, chatbot, chatbot, api_name="bot_response")
    bot_msg.then(lambda: gr.MultimodalTextbox(interactive=True), None, [chat_input])

    chatbot.like(print_like_dislike, None, None, like_user_message=True)

if __name__ == "__main__":
    demo.launch()
```

### Screenshot

![image](https://github.com/user-attachments/assets/77c3b6d7-ee42-475f-a0f3-69a3459a0feb)

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Linux
gradio version: 5.0.1
gradio_client version: 1.4.0

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 23.2.1
anyio: 4.4.0
fastapi: 0.112.2
ffmpy: 0.4.0
gradio-client==1.4.0 is not installed.
httpx: 0.27.2
huggingface-hub: 0.25.2
jinja2: 3.1.4
markupsafe: 2.1.5
numpy: 1.26.4
orjson: 3.10.7
packaging: 23.2
pandas: 2.2.2
pillow: 10.4.0
pydantic: 2.8.2
pydub: 0.25.1
python-multipart: 0.0.9
pyyaml: 6.0.2
ruff: 0.6.3
semantic-version: 2.10.0
tomlkit==0.12.0 is not installed.
typer: 0.12.5
typing-extensions: 4.12.2
urllib3: 2.2.2
uvicorn: 0.30.6
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2024.6.1
httpx: 0.27.2
huggingface-hub: 0.25.2
packaging: 23.2
typing-extensions: 4.12.2
websockets: 12.0
```

### Severity

Blocking usage of gradio

**Comment by kunzeng-ch** [comment]
Is your problem solved? I also encountered the same problem. When I enable the Gradio client, it gets stuck after typing and keeps timing.

**Comment by abidlabs** [comment]
Hi folks, we are looking into it! In the meantime, can you try: `file_types=["images", ".webp"]`

cc @dawoodkhan82

**Comment by skye0402** [comment]
> file_types=["images", ".webp"]

Yes, that seems to be a workaround. I figured that e.g. `.jpeg` files are also raising this error while `.jpg` doesn't. 😅

**Comment by abidlabs** [comment]
Hmm interesting, let us put together a fix

**Comment by xdtkyx010** [comment]
me too
IMPORTANT: You are using gradio version 4.44.1, however version 5.0.1 is available, please upgrade.

**Comment by dawoodkhan82** [comment]
I can't reproduce the issue. Perhaps it was fixed by this PR (https://github.com/gradio-app/gradio/pull/9678). @skye0402 Can you install the latest version of gradio 5.1.0 and see if you still have the issue?

**Comment by skye0402** [comment]
> I can't reproduce the issue. Perhaps it was fixed by this PR (#9678). @skye0402 Can you install the latest version of gradio 5.1.0 and see if you still have the issue?

@dawoodkhan82 I can't reproduce it with 5.1.0. So I consider it fixed. Closing the issue. Thank you!

**Comment by ewwink** [comment]
This is happen for Python 3.10 or older because the standar library **Mimetypes** has no `.webp`

https://github.com/python/cpython/blob/8773554b717cfb08b4bd11a927813f4ed74762c7/Lib/mimetypes.py#L517

while 3.11 and newer have it

https://github.com/python/cpython/blob/65da5db28a39cb0d9ee311206bdc8745b482c56f/Lib/mimetypes.py#L559

but for Windows 10 upgrading python to 3.13 not solve the problem because Windows 10 registry has no `image/webp` so the value `types_map[ext]` for  `.webp = 'image/webp'` overwritten into webp is `.webp = ''` 

https://github.com/python/cpython/blob/65da5db28a39cb0d9ee311206bdc8745b482c56f/Lib/mimetypes.py#L173
